_data/terms.yml

```diff
@@ -40,12 +44,15 @@
  "Mineral resource potential": "According to the U.S. Geological Survey, mineral resource potential is the likelihood for the occurrence of undiscovered mineral resources in a defined area."
  "Multi-Stakeholder Group (MSG)": "A cross-sector body comprised of members and alternates from government, industry, and civil society organizations commissioned by the Secretary of the Interior to guide and monitor EITI implementation."
  "North American Industry Classification System (NAICS)": "The standard used by federal agencies in classifying business establishments for the purpose of collecting, analyzing, and publishing statistical data related to the U.S. economy."
+ "ONRR": "The Office of Natural Resources Revenue (ONRR) is part of the U.S. Department of the Interior, and is responsible for collecting, disbursing, and verifying federal and Indian energy and other natural resource revenue."
  "Operating fee": "A fee for a percentage of the anticipated value of wind energy produced on federal waters."
  "Outer Continental Shelf": "The part of the continental shelf under federal jurisdiction, seaward of the line that marks state ownership, often three miles off a state’s coastline."
+ "OSMRE": "The Office of Surface Mining Reclamation and Enforcement (OSMRE) is part of the U.S. Department of the Interior, and is responsible for regulating surface coal mining in the United States, as well as funding the restoration of abandoned coal mines."
```

Comment:
I decided to break from our standard treatment — despite the inconsistency — because these are explicitly *acronym* definitions, and I want to make sure these come up quickly and recognizably when people search for the acronym. I should go make sure this is documented in our content guide, though!